package.json

```diff
@@ -83,7 +81,23 @@
     "d3-hexbin": "^0.2.2",
     "d3-request": "^1.0.6",
     "d3-scale": "^1.0.6",
-    "deck.gl": "~7.1.5",
+    "kepler-outdated-nebula.gl": "https://github.com/EQWorks/nebula.gl/raw/build/custom/modules/main/dist.tgz",
+    "kepler-outdated-nebula.gl-layers": "https://github.com/EQWorks/nebula.gl/raw/build/custom/modules/layers/dist.tgz",
+    "kepler-outdated-nebula.gl-edit-modes": "https://github.com/EQWorks/nebula.gl/raw/build/custom/modules/edit-modes/dist.tgz",
+    "kepler-outdated-luma.gl": "https://github.com/EQWorks/luma.gl/raw/build/custom/modules/main/dist.tgz",
+    "kepler-outdated-luma.gl-addons": "https://github.com/EQWorks/luma.gl/raw/build/custom/modules/addons/dist.tgz",
+    "kepler-outdated-luma.gl-constants": "https://github.com/EQWorks/luma.gl/raw/build/custom/modules/constants/dist.tgz",
+    "kepler-outdated-luma.gl-shadertools": "https://github.com/EQWorks/luma.gl/raw/build/custom/modules/shadertools/dist.tgz",
+    "kepler-outdated-luma.gl-webgl": "https://github.com/EQWorks/luma.gl/raw/build/custom/modules/webgl/dist.tgz",
+    "kepler-outdated-luma.gl-webgl-state-tracker": "https://github.com/EQWorks/luma.gl/raw/build/custom/modules/webgl-state-tracker/dist.tgz",
+    "kepler-outdated-luma.gl-webgl2-polyfill": "https://github.com/EQWorks/luma.gl/raw/build/custom/modules/webgl2-polyfill/dist.tgz",
+    "kepler-outdated-deck.gl": "https://github.com/EQWorks/deck.gl/raw/build/custom/modules/main/dist.tgz",
+    "kepler-outdated-deck.gl-core": "https://github.com/EQWorks/deck.gl/raw/build/custom/modules/core/dist.tgz",
+    "kepler-outdated-deck.gl-layers": "https://github.com/EQWorks/deck.gl/raw/build/custom/modules/layers/dist.tgz",
+    "kepler-outdated-deck.gl-geo-layers": "https://github.com/EQWorks/deck.gl/raw/build/custom/modules/geo-layers/dist.tgz",
+    "kepler-outdated-deck.gl-mesh-layers": "https://github.com/EQWorks/deck.gl/raw/build/custom/modules/mesh-layers/dist.tgz",
+    "kepler-outdated-deck.gl-aggregation-layers": "https://github.com/EQWorks/deck.gl/raw/build/custom/modules/aggregation-layers/dist.tgz",
+    "kepler-outdated-deck.gl-react": "https://github.com/EQWorks/deck.gl/raw/build/custom/modules/react/dist.tgz",
```

Comment:
similar comment as https://github.com/EQWorks/snoke/pull/1030#discussion_r760452535, try to install from the source instead of manually built packs